iceoryx_examples/iceperf/README.md

```diff
@@ -164,3 +167,125 @@ Wich technologies are measured depends on the operating system (e.g. no message
     Waiting for subscriber to unsubscribe ... 
     Finished!
 
+## Code walkthrough
+
+Here we roughly describe the setup for performing the measurements. Things like initialization, sending and receiving of data are technology specific and can be found in the respective files (e.g. uds.cpp for 
+unix doamin socket). Our focus here is on the abstraction layer on top which allows us to add new IPC tecehnologies or you to extend and compare with whatever.
```

Comment:
typo `technologies `

Reply:
Oh, just realized that my typo commit was not pushed